includes/wikia/services/VideoService.class.php

```diff
@@ -49,9 +52,21 @@ public function addVideo( $url ) {
 				$file = wfFindFile( $videoTitle );
 			}
 
-			// Add a default description if available and one doesn't already exist
-			$vHelper = new VideoHandlerHelper();
-			$vHelper->addDefaultVideoDescription( $file );
+			if ( !method_exists( $file, 'getTitle' ) ) {
```

Comment:
you're checking if getTitle method exists but it is not called here anywhere
should we check if file is instance of File class maybe?